source/manuals/logit-io-joiners-and-leavers.html.md.erb

```diff
@@ -0,0 +1,28 @@
+---
+title: Logit.io joiners and leavers processes
+---
+
+# <%= current_page.data.title %>
+
+This page documents how to get a logit.io account, and how to manage
+leavers.
+
+## New users
+
+Logit.io is available via GMail.  To log in: go to the Google Apps
+Marketplace (the menu in the top-right), scroll to the bottom, and
+click on the "Logit" icon.  The first time you do this, your account
+will be automatically created.
+
+Once your user account has been created, you need to be added to a
+team to be able to access particular logging stacks.  Your tech lead
+should be a logit administrator and be able to add you to your team;
+if they are not, or they are not available, you can ask for help in
+`#reliability-eng` on slack.
+
+## Leavers
+
+When people leave GDS and their G-Suite account is cancelled, they
+will also lose access to logit.  However, their account will still
+exist and need to be cleaned up.
```

Comment:
needs

Reply:
hmm, i used "need" to agree with "will" ie "will need to be cleaned up" but i think this is a sign that it should be rewritten to be simpler 😄